spec/app/domain/authentication/security/extract_reques_remote_addr_info_spec.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Authentication::Security::ExtractRequestRemoteAddrInfo do
```

Comment:
Add tests of :
1. duplication proxies ip address in the XFF
2. duplication of the first IP (the client) in XFF

Reply:
There is a test for duplication of IPs.